src/components/pages/IngredientPage.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FunctionComponent, useEffect, useState, useCallback } from 'react';
+import { Ingredient } from '../../models/ingredient.model';
+import ingredientService from '../../services/ingredient.service';
+import IngredientList from '../layout/IngredientList';
+import IngredientForm from '../layout/IngredientForm';
+import './IngredientPage.scss';
+
+const IngredientPage: FunctionComponent = () => {
+    const [isLoading, setLoading] = useState(false);
+    const [ingredients, setIngredients] = useState<Ingredient[]>([]);
+    const [error, setError] = useState("");
+    const [selctedIngredient, setSelectedIngredient] = useState<Ingredient>();
```

Comment:
Typo `selctedIngredient ` => `selectedIngredient `
```suggestion
const [selectedIngredient, setSelectedIngredient] = useState<Ingredient>();
```